tests/lms/services/exceptions_test.py

```diff
@@ -200,5 +200,3 @@ def _requests_exception(**kwargs):
             response.raise_for_status()
         except requests.RequestException as err:
             return err
-
-        assert False, "We should never get here"
```

Comment:
It looks like the original purpose of this was to provide a more obvious error if for some reason the code above failed to generate the expected exception. Is that not useful now?

Reply:
The line is never reached, so this just messes up the coverage report!

Reply:
Can't you tell the code coverage generator to ignore that line?

Reply:
Ah, yes let's do that instead. Changed